docs/mkdocs/docs/technical/contributing.md

```diff
@@ -1,19 +1,88 @@
-Setting up your OS dependencies
-===============================
+Docker Quickstart
+=================
 
-ManyLinux Docker image
-----------------------
-ManyLinux(https://github.com/pypa/manylinux) captures the common denominator of many Linux distributions,
-giving the confidence that code worked in this image runs everywhere.
+#### 1) Start the ArcticDB build docker image
 
-We [extend](https://github.com/man-group/ArcticDB/blob/master/build_tooling/build_many_linux_image.sh) it with
-additional dependencies for compilation purposes.
-You can download the output [here](https://github.com/man-group/ArcticDB/pkgs/container/cibuildwheel_manylinux).
+```bash
+docker pull ghcr.io/man-group/cibuildwheel_manylinux:2.12.1-3a897
+docker run -it ghcr.io/man-group/cibuildwheel_manylinux:2.12.1-3a897
+```
+
+> :warning: The below instructions do not have to be run in the provided docker image. They can be run against any Python installation on any Linux distribution as long as the basic build [dependencies are available](#setting-up-linux).
+> 
+> If running outside the provided docker image, please change `/opt/python/cp39-cp39/bin/python3` in the examples below 
+> to an appropriate path for Python.
+
+#### 2) Check out ArcticDB including submodules
+
+```bash
+cd 
+git clone https://github.com/man-group/ArcticDB.git
+cd ArcticDB
+git submodule init && git submodule update
+```
+
+#### 3) Kick off the build
+
+```bash
+/opt/python/cp39-cp39/bin/python3 -m pip install -U pip setuptools wheel grpcio-tools
+ARCTIC_CMAKE_PRESET=skip /opt/python/cp39-cp39/bin/python3 setup.py develop
```

Comment:
For a quick start, it will be sufficient to run `pip install -ve .` (without `ARCTIC_CMAKE_PRESET=skip`).

The original version was trying to explain what above does internally which most users can then replace with IDE built-in features.

Reply:
Yeh I tried that. Couldn't get it to work for love nor money to be honest.

This does work and honestly I think is more descriptive.

Reply:
Discussed - appreciate `pip install -ve .` is one line but I think this is more descriptive and still can be copied pasted